### PR TITLE
[PT-BR] head-seo.ts and playground.ts

### PR DESCRIPTION
**Pull request by FelipeBit**

### CODE REVIEW

File: packages/typescriptlang-org/src/lib/release-info.json [hunk 1]
```diff
@@ -19,12 +19,12 @@
...
-    "rc": {
       "vs2017_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40rc",
       "vs2019_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40rc"
+    },
+    "rc": {
+      "vs2017_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40beta",
+      "vs2019_download": "https://marketplace.visualstudio.com/items?itemName=TypeScriptTeam.typescript-40beta"
     }
   }
 }
```
Comment (orta) [inline packages/typescriptlang-org/src/lib/release-info.json]: If you remove the changes to this file (I'm unsure what changed to be honest) tenths POR can be auto-merged by the folks running the Portuguese language

File: packages/typescriptlang-org/src/copy/pt/playground.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  play_sidebar_plugins_plugin_dev_copy:
+    "Tente conectar-se automaticamente a um plugin de playground no modo de desenvolvimento. Você pode começar a <a href='/dev/playground-plugins/' title='Link para a página sobre plugins do Playground'>fazer um plugin aqui</a>",
+  play_export_report_issue: "Relatar 'Issue' do TypeScript no GitHub",
+  play_export_copy_md: "Copiar Como 'Issue Markdown'",
+  play_export_copy_link: "Copiar Como 'Link Markdown'",
+  play_export_copy_link_preview: "Copiar como 'Link Markdown com visualização'",
+  play_export_tsast: "Abrir no 'TypeScript AST Viewer'",
+  play_export_sandbox: "Abrir no CodeSandbox",
+  play_export_stackblitz: "Abrir no StackBlitz",
+  play_export_clipboard: "[URL copiado para a área de transferência]",
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/playground.ts:48]:
```suggestion
  play_export_clipboard: "[URL copiada para a área de transferência]",
```

File: packages/typescriptlang-org/src/copy/pt/playground.ts [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  play_run_js: "JavaScript Executado",
+  play_run_ts: "TypeScript Transpilado Executado",
+  play_run_js_fail: "O JavaScript Executado Falhou:",
+  play_default_code_sample: `// Bem-vindo ao TypeScript Playground, este é um site
+// que lhe dá a chance de escrever, compartilhar e aprender TypeScript.
```
Comment (khaosdoctor) [inline packages/typescriptlang-org/src/copy/pt/playground.ts:53]:
```suggestion
// que te dá a chance de escrever, compartilhar e aprender TypeScript.
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Small changes

**Comment by khaosdoctor** [comment]
The PR also shows a conflicting file on `release-info.json`, did you change that or is that an automated change?

@orta @FelipeBit

**Comment by khaosdoctor** [comment]
@FelipeBit Can you remove the changes on the release-info please? So that we can merge over here

**Comment by FelipeBit** [comment]
@khaosdoctor of course, i will do it today.

**Comment by khaosdoctor** [comment]
Thanks a lot!

File: packages/typescriptlang-org/src/lib/release-info.json (hunk 1 above)
Comment (FelipeBit) [reply packages/typescriptlang-org/src/lib/release-info.json]: Adjusted

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**